specs/referrer-policy/index.src.html

```diff
@@ -516,18 +516,18 @@ <h3 id="referrer-policy-delivery-meta">Delivery via <{meta}></h3>
   </dl>
 
   <h3 id="referrer-policy-delivery-referrer-attribute">Delivery
-  via a <code>referrer</code> attribute</h3>
+  via a <code>referrerpolicy</code> attribute</h3>
 
-  A referrer policy may be set when a <code>referrer</code> attribute is
+  A referrer policy may be set when a <code>referrerpolicy</code> attribute is
```

Comment:
I don't think either of those should really be affected by this attribute on an `<a>`. That would be way too magical.